py/surveyqa/nightly.py

```diff
@@ -194,10 +211,16 @@ def get_skypathplot(exposures, tiles, width=600, height=300):
     obs = fig.scatter('RA', 'DEC', size=5, fill_alpha=0.7, legend='PROGRAM', source=src, color=mapper)
     fig.line(src.data['RA'], src.data['DEC'], color='navy', alpha=0.4)
     
-    #- Circles the first point observed on NIGHT
+    #- Stars the first point observed on NIGHT
     first = tiles_and_exps[0]
     fig.asterisk(first['RA'], first['DEC'], size=10, line_width=1.5, fill_color=None, color='gold')
     
+    #- Adds moon location at midnight on NIGHT
+    mjd_midnight = int(exposures['MJD'][0])
+    moon_loc = get_moonloc(mjd_midnight)
+    ra, dec = float(moon_loc.ra.to_string(decimal=True)), float(moon_loc.dec.to_string(decimal=True))
+    fig.circle(ra, dec, size=10, color='black')
```

Comment:
Minor: let's make the moon color "gold" and pick a different color for the start-of-night star symbol.